README.md

```diff
@@ -1,7 +1,30 @@
-# Recoil
+# Recoil &middot; [![Node.js CI](https://github.com/facebookexperimental/Recoil/workflows/Node.js%20CI/badge.svg)](https://github.com/facebookexperimental/Recoil/actions) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/facebookexperimental/Recoil/blob/master/LICENSE)
 
-[![Node.js CI](https://github.com/facebookexperimental/Recoil/workflows/Node.js%20CI/badge.svg)](https://github.com/facebookexperimental/Recoil/actions)
+Recoil is an experimental set of utilities for state management with React.
 
-Recoil is an experimental set of utilities for state management with React. Please see the website: https://recoiljs.org
+Please see the website: https://recoiljs.org
 
-Please see the LICENSE file for the MIT license under which Recoil is provided.
+## Installation
+
+The Recoil package lives in [npm](https://www.npmjs.com/get-npm). To install the latest stable version, run the following command:
+
+```shell
+npm install recoil
+```
+
+Or if you're using <a href="https://classic.yarnpkg.com/en/docs/install/" target="_blank">yarn</a>:
```

Comment:
For consistency with the npm link maybe this should also use markdown style links
`[yarn](https://classic.yarnpkg.com/en/docs/install/)`